src/core/IronPython.Modules/termios.cs

```diff
@@ -0,0 +1,383 @@
+
+    #region Other Public Constants
+    // Linux: glibc/bits/termios.h (/usr/include/{x86_64,aarch64}-linux-gnu/)
+    // macOS: usr/include/sys/termios.h (/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk)
+
+    // iflag
+    public const int IGNBRK  = 0x0001;    // ignore break condition
+    public const int BRKINT  = 0x0002;    // signal interrupt on break
+    public const int IGNPAR  = 0x0004;    // ignore characters with parity errors
+    public const int PARMRK  = 0x0008;    // mark parity and framing errors
+    public const int INPCK   = 0x0010;    // enable input parity check
+    public const int ISTRIP  = 0x0020;    // mask off 8th bit
+    public const int INLCR   = 0x0040;    // map NL into CR on input
+    public const int IGNCR   = 0x0080;    // ignore CR
+    public const int ICRNL   = 0x0100;    // map CR to NL on input
+    public static int IXON   => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ?
+                               0x0200 : 0x0400;    // enable start output control
+    public static int IXOFF  => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ?
+                               0x0400 : 0x1000;    // enable stop output control
+    public const int IXANY   = 0x0800;    // any char will restart after stop
+    public const int IMAXBEL = 0x2000;   // ring bell on input queue full
+    public const int IUTF8   = 0x4000;     // maintain state for UTF-8 VERASE
+
+
+    // oflag
+    public const  int OPOST  = 0x0001;    // enable output processing
+    public static int ONLCR => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ?
+                              0x0002 : 0x0004; // map NL to CR-NL
+
+
+    // cflag
+    public static int CSIZE  => CS5 | CS6 | CS7 | CS8; // number of bits per character
+    public static int CS5    =>                                                   0x0000;           // 5 bits per character
+    public static int CS6    => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x0100 : 0x0010;  // 6 bits per character
+    public static int CS7    => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x0200 : 0x0020;  // 7 bits per character
+    public static int CS8    => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x0300 : 0x0030;  // 8 bits per character
+    public static int CREAD  => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x0800 : 0x0080;  // enable receiver
+    public static int PARENB => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x1000 : 0x0100;  // parity enable
+    public static int HUPCL  => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x4000 : 0x0400;  // hang up on last close
+
+    [PythonHidden(PlatformID.MacOSX)]
+    public static int CBAUD  => 0x100f;  // mask for baud rate
+    [PythonHidden(PlatformID.MacOSX)]
+    public static int CBAUDEX => 0x1000;  // extra baud speed mask
+
+
+    // lflag
+    public static uint ECHOKE  => RuntimeInformation.IsOSPlatform(OSPlatform.OSX) ? 0x0001u : 0x0800u;       // visual erase for line kill
```

Comment:
I guess these will show up as `System.UInt32` instead of `int` in the Python world. Not necessarily a big deal since this module is somewhat obscure, but something to keep in mind...

Reply:
Yes, I was debating which type to use here. The issue with `BigInteger` is that it is not very efficient at storing values that don't fit in 32 bits. Here, all those constants are flags and are supposed be used as opaque values, evt. to be OR-ed together, not for some complex integer arithmetics (although it would still be OK, IronPython's support for mixed type integer arithmetics and automatic type conversions is quite extensive). The API that accepts them is/will be using `long` so that it will efficiently capture `UIn32` and negative values of `In32`. Yes, something to keep in mind, though I don't expect it to be a problem.